aws_lambda_powertools/logging/logger.py

```diff
@@ -318,12 +318,12 @@ def set_correlation_id(self, value: str):
         self.structure_logs(append=True, correlation_id=value)
 
     @staticmethod
-    def _get_log_level(level: Union[str, int]) -> Union[str, int]:
+    def _get_log_level(level: Union[str, int, None]) -> Union[str, int]:
         """ Returns preferred log level set by the customer in upper case """
         if isinstance(level, int):
             return level
 
-        log_level: str = level or os.getenv("LOG_LEVEL")
+        log_level: Optional[str] = level or os.getenv("LOG_LEVEL")
```

Comment:
Shouldn't this be 

```suggestion
        log_level: Union[str, int, None] = level or os.getenv("LOG_LEVEL")
```

Reply:
@heitorlessa  i can't be an int at this point as we already exist early before this.

Reply:
ie: 
```python3
if isinstance(level, int):
```

Reply:
missed it despite bring right in front of me.